src/tests/Parser/DisciplinePriest/Features/AtonementAttribution.test.js

```diff
@@ -0,0 +1,30 @@
+import AtonementAttribution from 'Parser/DisciplinePriest/Modules/Features/AtonementAttribution';
+import SPELLS from 'common/SPELLS';
+import processEvents from '../Fixtures/processEvents';
+
+import {
+  TwoSuccesiveDamagingEventsWithAtonementOnSelfBetween,
+  TwoSuccesiveDamagingEvents
+} from '../Fixtures/ReorderingScenarios';
+
+describe('DisciplinePriest.Reordering', () => {
+
+  let atonementAtribution;
+  beforeEach(() => {
+    atonementAtribution = new AtonementAttribution({
+      reorder: () => true,
+      toPlayer: () => true,
+      byPlayer: () => true,
+      toPlayerPet: () => false,
+      byPlayerPet: () => false,
+    });
+  });
+
+  it('Two Succesive Damaging Events with atonement on self between is reordered', () => {
```

Comment:
It's unclear to me what the end-goal is from this. Would this be right: `the Atonement self-healing triggered by two successive damaging events gets grouped with the rest of the Atonement healing`?

Reply:
Correct